doc/languages-frameworks/rust.section.md

```diff
@@ -116,22 +116,44 @@ is updated after every change to `Cargo.lock`. Therefore,
 a `Cargo.lock` file using the `cargoLock` argument. For example:
 
 ```nix
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
```

Comment:
`rec` is not needed here and could confuse the reader. Not really related but I let's keep this snippet and the new, very similar one below in sync.